bazel-mode.el

```diff
@@ -21,9 +21,20 @@
 ;;
 ;;; Code:
 
+(defconst bazel-mode-syntax-table
+  (let ((table (make-syntax-table)))
+	; single and multiple line comment start
+	(modify-syntax-entry ?# "< 14" table)
+	; single line comment end
+	(modify-syntax-entry ?\n ">" table)
+	table)
+  "Syntax table for `bazel-mode'.")
+
 ;;;###autoload
 (define-derived-mode bazel-mode prog-mode "Bazel"
-  "Major mode for editing Bazel BUILD and WORKSPACE files.")
```

Comment:
Please also set the variables `comment-start`, `comment-end`, `comment-start-skip`, and `comment-use-syntax` here.

Reply:
Done.